multinet/api.py

```diff
@@ -138,72 +138,48 @@ def delete_workspace(workspace: str) -> Any:
 
 
 @bp.route("/workspaces/<workspace>/graphs/<graph>", methods=["POST"])
-@use_kwargs({"node_tables": fields.List(fields.Str()), "edge_table": fields.Str()})
+@use_kwargs({"edge_table": fields.Str()})
 @swag_from("swagger/create_graph.yaml")
-def create_graph(
-    workspace: str,
-    graph: str,
-    node_tables: Optional[List[str]] = None,
-    edge_table: Optional[str] = None,
-) -> Any:
+def create_graph(workspace: str, graph: str, edge_table: Optional[str] = None) -> Any:
     """Create a graph."""
 
-    if not node_tables or not edge_table:
-        body = request.data.decode("utf8")
-        raise MalformedRequestBody(body)
-
-    missing = [
-        arg[0]
-        for arg in [("node_tables", node_tables), ("edge_table", edge_table)]
-        if arg[1] is None
-    ]
-    if missing:
-        raise RequiredParamsMissing(missing)
+    if not edge_table:
+        raise RequiredParamsMissing(["edge_table"])
 
     loaded_workspace = db.db(workspace)
     if loaded_workspace.has_graph(graph):
         raise AlreadyExists("Graph", graph)
 
-    existing_tables = set([x["name"] for x in loaded_workspace.collections()])
-    edges = loaded_workspace.collection(edge_table).all()
+    edge_table_properties = util.get_edge_table_properties(workspace, edge_table)
+    node_tables = edge_table_properties["table_keys"]
+    from_tables = edge_table_properties["from_tables"]
+    to_tables = edge_table_properties["to_tables"]
```

Comment:
Could you add a comment to this block of code explaining what the util function is doing?